tracing-core/src/event.rs

```diff
@@ -94,6 +94,11 @@ impl<'a> Event<'a> {
         self.fields.field_set().iter()
     }
 
+    /// Returns the field values on this `Event`.
+    pub fn values(&self) -> &field::ValueSet {
```

Comment:
I think we should try to avoid changing `tracing-core` unless it's absolutely necessary. I think we can achieve the goal of serializing the fields as a nested object as discussed in https://github.com/tokio-rs/tracing/pull/377#discussion_r334118874 without having to make changes in core. I've opened https://github.com/pimeys/tracing/pull/1 to propose one potential solution — feel free to merge it into your branch, or come up with something else.